Formula/gajim.rb

```diff
@@ -0,0 +1,192 @@
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/source/s/six/six-1.12.0.tar.gz"
+    sha256 "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73"
+  end
+
+  resource "Pillow" do
+    url "https://files.pythonhosted.org/packages/source/P/Pillow/Pillow-5.4.1.tar.gz"
+    sha256 "5233664eadfa342c639b9b9977190d64ad7aca4edc51a966394d7e08e7f38a9f"
+  end
+
+  resource "python-gnupg" do
+    url "https://files.pythonhosted.org/packages/source/p/python-gnupg/python-gnupg-0.4.3.tar.gz"
+    sha256 "2d158dfc6b54927752b945ebe57e6a0c45da27747fa3b9ae66eccc0d2147ac0d"
+  end
+
+  resource "python-axolotl" do
+    url "https://files.pythonhosted.org/packages/source/p/python-axolotl/python-axolotl-0.1.42.tar.gz"
+    sha256 "ef78c2efabcd4c33741669334bdda04710a3ef0e00b653f00127acff6460a7f0"
+  end
+
+  resource "python-axolotl-curve25519" do
+    url "https://files.pythonhosted.org/packages/source/p/python-axolotl-curve25519/python-axolotl-curve25519-0.4.1.post2.tar.gz"
+    sha256 "0705a66297ebd2f508a60dc94e22881c754301eb81db93963322f6b3bdcb63a3"
+  end
+
+  resource "qrcode" do
+    url "https://files.pythonhosted.org/packages/source/q/qrcode/qrcode-6.0.tar.gz"
+    sha256 "037b0db4c93f44586e37f84c3da3f763874fcac85b2974a69a98e399ac78e1bf"
+  end
+
+  resource "pybonjour" do
+    url "https://dev.gajim.org/lovetox/pybonjour-python3/-/archive/pybonjour-1.1.2/pybonjour-python3-pybonjour-1.1.2.tar.gz"
+    sha256 "50b2459e5410d344e1f8ed2c2d3ab0d81750b008a82477edba5c250c1d592187"
+  end
+
+  # some popular plugins
+
+  resource "plugin_installer" do
+    url "https://ftp.gajim.org/plugins_1.1_zip/plugin_installer.zip"
+    sha256 "62f3c84060b4cc0f03bbc6b509b6db83831fa6f4ff977640f712a055424c2064"
+  end
+
+  resource "omemo" do
+    url "https://ftp.gajim.org/plugins_1.1_zip/omemo.zip"
+    sha256 "4f7bccd0f109ec685722bce6a72eb7f5f12e0cb381c8194af0d504429af7c9f7"
+  end
+
+  resource "url_image_preview" do
+    url "https://ftp.gajim.org/plugins_1.1_zip/url_image_preview.zip"
+    sha256 "45a913f799ac6eff435fb738d95d6d2690c4a981704ee7b43c248b3d7b56984e"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+
+    resource("Pillow").stage do
+      inreplace "setup.py" do |s|
+        sdkprefix = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
+        s.gsub! "openjpeg.h", "probably_not_a_header_called_this_eh.h"
+        s.gsub! "ZLIB_ROOT = None", "ZLIB_ROOT = ('#{sdkprefix}/usr/lib', '#{sdkprefix}/usr/include')"
+        s.gsub! "JPEG_ROOT = None", "JPEG_ROOT = ('#{Formula["jpeg"].opt_prefix}/lib', '#{Formula["jpeg"].opt_prefix}/include')"
+        s.gsub! "FREETYPE_ROOT = None", "FREETYPE_ROOT = ('#{Formula["freetype"].opt_prefix}/lib', '#{Formula["freetype"].opt_prefix}/include')"
+      end
+
+      ENV.append "CFLAGS", "-I#{MacOS.sdk_path}/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers" unless MacOS::CLT.installed?
```

Comment:
This should do the trick:
```suggestion
    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
```

Reply:
rationale: the bindings cannot be compiled at all with the CLT IIRC.